index.html

```diff
@@ -99,6 +109,93 @@ <h2>Multi-Post <br> Stories </h2>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="sec-list">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="desk-second-div">
+            <div class="div-info">
+                <h2>Multi-Post Stories</h2>
+                <ul class="ul-li">
+                    <li>FACEBOOK</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>Full Stack Dev</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+                <p>Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+                <ul class="sec-list">
+                    <li>html</li>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="desk-div-img"><img src="images/Snapshoot Portfolio (2).png"></div>
+        </div>
+        <div class="desk-third-div">
+            <div class="desk-div-img"><img src="images/Snapshoot Portfolio (3).png"></div>
+            <div class="div-info">
+                <h2>Facebook 360</h2>
+                <ul class="ul-li">
+                    <li>FACEBOOK</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>Full Stack Dev</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+                <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+                <ul class="sec-list">
+                    <li>html</li>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="desk-fourth-div">
+            <div class="desk-div-img"><img src="images/Snapshoot Portfolio.png"></div>
+            <div class="div-info">
+                <h2>Uber Navigation</h2>
+                <ul class="ul-li">
+                    <li>Uber</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="images/Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+                <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
```

Comment:
- [ ] Please ☺️  make sure the `<p>` paragraph content of `work-sec` cards looks just like that in [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z0umLtHjW2jvzRBK-0).  I recommend removing  `text-align: justify;` for the mobile version

**_Figma_**

<img width="176" alt="Screenshot 2023-02-23 at 10 43 49 PM" src="https://user-images.githubusercontent.com/44798044/220987477-a9c3833d-8bbb-4e93-9c67-029c15c81944.png">

**_Yours_**

<img width="312" alt="Screenshot 2023-02-23 at 9 22 54 PM" src="https://user-images.githubusercontent.com/44798044/220987363-34a73dcc-946d-4101-849f-2f9b4323b4d2.png">